blog_post/dbt_blog_post.md

```diff
@@ -0,0 +1,163 @@
+        columns:
+            -   name: transactionid
+            -   name: userid
+                meta:
+                    pace_transforms:
+                        # The administrator gets the original userid (identity).
+                        -   principals: [ { group: administrator } ]
+                            identity: { }
+                        # Everyone else gets a null value.
+                        -   principals: [ ]
+                            nullify: { }
+            -   name: email
+                meta:
+                    pace_transforms:
+                        # Again, the administrator gets the original value.
+                        -   principals: [ { group: administrator } ]
+                            identity: { }
+                        # Everyone else gets a partially masked email address using a regular 
+                        # expression transform.
+                        -   principals: [ ]
+                            regexp:
+                                regexp: "^.*(@.*)$"
+                                replacement: "****$1"
+            -   name: age
+            -   name: transactionamount
+            -   name: brand
+        meta:
+            pace_rule_sets:
+                -   filters:
+                        -   generic_filter:
+                                conditions:
+                                    # The "true" condition results in every row being shown to
+                                    # the administrators.
+                                    -   principals: [ { group: administrator } ]
+                                        condition: "true"
+                                    # Everyone else only sees records where the age is greater than 18
+                                    -   principals: [ ]
+                                        condition: "age > 18"
+```
+
+For a dbt project using databricks as data platform, the PACE dbt module will generate the following
+dbt model:
+
+```sql
+{#
+    This file was auto-generated by PACE. Do not edit this file directly.
+#}
+{{
+    config(
+      materialized='view',
+      meta={'pace_generated': true}
+    )
+}}
+
+select transactionid,
+       case
+           when (is_account_group_member('administrator')) then userid
+           else null
+           end userid,
+       case
+           when (is_account_group_member('administrator')) then email
+           else regexp_replace('email', '^.*(@.*)$', '****$1')
+           end email,
+       age,
+       transactionamount,
+       brand
+from {{ ref('dim_transactions') }}
+where case
+          when (is_account_group_member('administrator')) then true
+          else age > 18
+          end
+
+```
+
+In this example we explicitly specified the field transforms. PACE server supports tag-based
+transforms, let us know if you'd like to have those for PACE dbt too!
+
+[//]: # (Todo: call to action: try it out, provide feedback, join Slack)
```

Comment:
Yes, I was going to say it ends a bit abruptly. Suggestions:
- Link to beta tester vacancy
- Join slack
- Discuss with us @ ... 
- Try it out (`git clone`, `brew install` etc, or just a link to the quickstart.